internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/TestActiveActiveReplicationForIncPush.java

```diff
@@ -207,4 +208,101 @@ public void testAAReplicationForIncrementalPushToRT() throws Exception {
     }
   }
+
+  @Test
+  public void testAggregateModeForIncrementalPushToRT() throws Exception {
+    String clusterName = CLUSTER_NAMES[0];
+    File inputDirBatch = getTempDataDirectory();
+    File inputDirInc1 = getTempDataDirectory();
+    File inputDirInc2 = getTempDataDirectory();
+
+    String parentControllerUrls =
+        parentControllers.stream().map(VeniceControllerWrapper::getControllerUrl).collect(Collectors.joining(","));
+    String inputDirPathBatch = "file:" + inputDirBatch.getAbsolutePath();
+    String inputDirPathInc1 = "file:" + inputDirInc1.getAbsolutePath();
+    String inputDirPathInc2 = "file:" + inputDirInc2.getAbsolutePath();
+    Function<Integer, String> connectionString = i -> childDatacenters.get(i).getControllerConnectString();
+
+    try (ControllerClient parentControllerClient = new ControllerClient(clusterName, parentControllerUrls);
+        ControllerClient dc0ControllerClient = new ControllerClient(clusterName, connectionString.apply(0));
+        ControllerClient dc1ControllerClient = new ControllerClient(clusterName, connectionString.apply(1));
+        ControllerClient dc2ControllerClient = new ControllerClient(clusterName, connectionString.apply(2))) {
+      String storeName = Utils.getUniqueString("store");
+      Properties propsBatch = defaultVPJProps(parentControllerUrls, inputDirPathBatch, storeName);
+      propsBatch.put(SEND_CONTROL_MESSAGES_DIRECTLY, true);
+      Properties propsInc1 = defaultVPJProps(parentControllerUrls, inputDirPathInc1, storeName);
+      propsInc1.put(SEND_CONTROL_MESSAGES_DIRECTLY, true);
+      Properties propsInc2 = defaultVPJProps(parentControllerUrls, inputDirPathInc2, storeName);
+      propsInc2.put(SEND_CONTROL_MESSAGES_DIRECTLY, true);
+
+      Schema recordSchema = TestPushUtils.writeSimpleAvroFileWithUserSchema(inputDirBatch, true, 100);
+      String keySchemaStr = recordSchema.getField(VenicePushJob.DEFAULT_KEY_FIELD_PROP).schema().toString();
+      String valueSchemaStr = recordSchema.getField(VenicePushJob.DEFAULT_VALUE_FIELD_PROP).schema().toString();
+
+      propsInc1.setProperty(INCREMENTAL_PUSH, "true");
+      propsInc1.put(SOURCE_GRID_FABRIC, "dc-2");
+      TestPushUtils.writeSimpleAvroFileWithUserSchema2(inputDirInc1);
+
+      propsInc2.setProperty(INCREMENTAL_PUSH, "true");
+      propsInc2.put(SOURCE_GRID_FABRIC, "dc-2");
+      TestPushUtils.writeSimpleAvroFileWithUserSchema3(inputDirInc2);
+
+      TestUtils.assertCommand(parentControllerClient.createNewStore(storeName, "owner", keySchemaStr, valueSchemaStr));
+      // Store Setup
+      UpdateStoreQueryParams updateStoreParams =
+          new UpdateStoreQueryParams().setStorageQuotaInByte(Store.UNLIMITED_STORAGE_QUOTA)
+              .setPartitionCount(1)
+              .setHybridOffsetLagThreshold(2L)
+              .setHybridRewindSeconds(TEST_TIMEOUT / 2)
+              .setIncrementalPushEnabled(true)
```

Comment:
Did we deprecate incPushPolicy yet?  Maybe throw it in here just to be extra sure the test set up is doing the thing we expect it to.

Reply:
Yes, we did and hence no need to set it explicitly.